framework/include/timesteppers/CompositionDT.h

```diff
@@ -57,7 +107,7 @@ class CompositionDT : public TimeStepper
   /**
    * Internal method for querying TheWarehouse for the currently active timesteppers.
    */
-  std::vector<TimeStepper *> getTimeSteppers();
+  std::vector<TimeStepper *> getTimeSteppers() const;
```

Comment:
You should very rarely make a const method that returns non const objects.

Where are you calling this that requires it to be const? If you really need this, you should have:

```
std::vector<TimeStepper *> getTimeSteppers();
std::vector<const TimeStepper *> getTimeSteppers() const;
```